src/platform/testing/e2e/nightwatch-commands/checkDisabledElement.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @method checkDisabledElement
+ * @param {string} [selector] The selector (CSS / Xpath) used to locate the element.
+ * @param {function} [callback] Optional callback function to be called when the command finishes.
+ * @returns {boolean}
+ * @api commands
+ */
+exports.command = function checkDisabledElement(selector, callback) {
```

Comment:
Is this command useful outside the context of the assertion? If not, consider moving inside the assertion function.

Reply:
Works much better this way!